src/tests/Auth/notFound.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { NotFound } from '../../App';
+
+describe('Not found Component', () => {
+  it('should render without crashing', () => {
+    expect(mount.bind(null, <NotFound />)).not.toThrow();
+  });
+});
```

Comment:
You could take a snapshot to be sure the component is properly rendered.

Reply:
Ok thanks